tools/coverage/presubmit.go

```diff
@@ -27,16 +27,19 @@ import (
 	"knative.dev/test-infra/tools/coverage/line"
 )
 
-func RunPresubmit(p *gcs.PreSubmit, arts *artifacts.LocalArtifacts) (isCoverageLow bool) {
+// RunPresubmit runs the pre-submit procedure
+func RunPresubmit(p *gcs.PreSubmit, arts *artifacts.LocalArtifacts, noRepoConn bool) (isCoverageLow bool) {
 	log.Println("starting PreSubmit.RunPresubmit(...)")
 	coverageThresholdInt := p.CovThreshold
 
-	concernedFiles := githubUtil.GetConcernedFiles(&p.GithubPr, "")
-
-	if len(*concernedFiles) == 0 {
-		log.Printf("List of concerned committed files is empty, " +
-			"don't need to run coverage profile in presubmit\n")
-		return false
+	var concernedFiles *map[string]bool
+	if !noRepoConn {
+		concernedFiles = githubUtil.GetConcernedFiles(&p.GithubPr, "")
+		if len(*concernedFiles) == 0 {
```

Comment:
```suggestion
		if len(githubUtil.GetConcernedFiles(&p.GithubPr, "")) == 0 {
```

Reply:
not applicable. concernedFiles used later